### DISCUSSION TITLE
Recurring Silences or "Business Hours Only" alerts

### DISCUSSION THREAD

**Discussion by reyjrar** [grafana/grafana#40900] [Alerting]
Would be great to either extend silences or allow alerts to be configured to only send notifications during certain times.

**Comment by JohnnyQQQQ** [comment]
We are tracking our progress for this feature here https://github.com/grafana/grafana/issues/37752

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
